lib/yard-sorbet/sig_to_yard.rb

```diff
@@ -95,4 +92,10 @@ def self.convert_type(node)
       [node.source]
     end
   end
+
+  def self.fix_generic_aref(node)
```

Comment:
I know I had proposed the `fix_generic_aref` name, but now I think it looks ugly. Maybe:
```suggestion
  def self.build_generic_type(node)
```
instead?